Makefile

```diff
@@ -99,6 +99,14 @@ test: reports
     -o log_cli=true -o log_level=INFO -o log_file=reports/tests.log \
     tests/* 2>&1 | tee reports/tests.log'
 
+.PHONY: test-custom
+test-custom: reports
+	@bash -c 'set -o pipefail; export PYTHONPATH=$(PWD); \
+	$(PIPENV) run pytest -s -v --junitxml=reports/junit.xml \
+	--import-mode importlib \
+    -o log_cli=true -o log_level=INFO -o log_file=reports/tests.log \
+    tests/src/datasets/test_text_dpo_modeling_ds.py --tb=no'
+
```

Comment:
Can you explain the purpose for this new make command, please?

Reply:
sure, to manually switch just single tests without full log outputs